package/whiteboard_hypercard/package

```diff
@@ -0,0 +1,31 @@
+# vim: set ft=sh:
+pkgname=whiteboard_hypercard
+pkgdesc="Real-time collaboration, drawing or whiteboarding"
+url=https://github.com/fenollp/reMarkable-tools
+pkgver=0.3.0
+timestamp=2020-09-21T17:59Z
+section=games
```

Comment:
Your package seems to fit in the same category as harmony, which will be moved to `drawing` soon (see #105).

```suggestion
section=drawing
```